envs/conda-env.yml

```diff
@@ -0,0 +1,132 @@
+  - googleapis-common-protos=1.52.0=py37hecd8cb5_0
+  - html5lib=1.1=pyh9f0ad1d_0
+  - httplib2=0.19.0=pyhd8ed1ab_0
+  - idna=2.10=pyhd3eb1b0_0
+  - intel-openmp=2019.4=233
+  - isodate=0.6.0=py_1
+  - keepalive=0.5=py37hf985489_5
+  - kubernetes=1.18.8=0
+  - kubernetes-client=1.18.8=h321dcc0_0
+  - kubernetes-node=1.18.8=h321dcc0_0
+  - kubernetes-server=1.18.8=h321dcc0_0
+  - libcrc32c=1.1.1=hb1e8313_2
+  - libcxx=10.0.0=1
+  - libedit=3.1.20191231=h1de35cc_1
+  - libffi=3.3=hb1e8313_2
+  - libprotobuf=3.14.0=h2842e9f_0
+  - libsodium=1.0.18=hbcb3906_1
+  - mkl=2019.4=233
+  - mkl-service=2.3.0=py37h9ed2024_0
+  - mkl_fft=1.3.0=py37ha059aab_0
+  - mkl_random=1.1.1=py37h959d312_0
+  - multidict=5.1.0=py37h9ed2024_2
+  - ncurses=6.2=h0a44026_1
+  - numpy=1.19.2=py37h456fd55_0
+  - numpy-base=1.19.2=py37hcfb5961_0
+  - oauthlib=3.1.0=py_0
+  - openjdk=8.0.152=h1de35cc_3
+  - openssl=1.1.1j=hbcf498f_0
+  - pandas=1.2.2=py37hb2f4e1b_0
+  - paramiko=2.7.2=pyh9f0ad1d_0
+  - pip=21.0.1=py37hecd8cb5_0
+  - protobuf=3.14.0=py37h23ab428_1
+  - py=1.10.0=pyhd3deb0d_0
+  - py4j=0.10.7=py37_0
+  - pyasn1=0.4.8=py_0
+  - pyasn1-modules=0.2.8=py_0
+  - pycparser=2.20=py_2
+  - pyjwt=2.0.1=py37hecd8cb5_0
+  - pynacl=1.4.0=py37hf967b71_2
+  - pyopenssl=20.0.1=pyhd3eb1b0_1
+  - pyparsing=2.4.7=pyh9f0ad1d_0
+  - pysftp=0.2.9=py_1
+  - pysocks=1.7.1=py37hecd8cb5_0
+  - pyspark=2.4.0=py37_0
+  - python=3.7.9=h26836e1_0
+  - python-dateutil=2.8.1=pyhd3eb1b0_0
+  - python_abi=3.7=1_cp37m
+  - pytz=2021.1=pyhd3eb1b0_0
+  - rdflib=5.0.0=py37hf985489_3
+  - readline=8.1=h9ed2024_0
+  - requests=2.25.1=pyhd3eb1b0_0
+  - requests-oauthlib=1.3.0=py_0
+  - retry=0.9.2=py_0
+  - rsa=4.7.2=pyhd3eb1b0_1
+  - setuptools=52.0.0=py37hecd8cb5_0
+  - simplejson=3.17.2=py37hf967b71_2
+  - six=1.15.0=py37hecd8cb5_0
+  - sparqlwrapper=1.8.5=py37hf985489_1005
+  - sqlite=3.33.0=hffcf06c_0
+  - tk=8.6.10=hb0a8c7a_0
+  - tqdm=4.58.0=pyhd8ed1ab_0
+  - uritemplate=3.0.1=py_0
+  - urllib3=1.26.3=pyhd3eb1b0_0
+  - webencodings=0.5.1=py_1
+  - wheel=0.36.2=pyhd3eb1b0_0
+  - xz=5.2.5=h1de35cc_0
+  - yarl=1.5.1=py37haf1e3a3_0
+  - zlib=1.2.11=h1de35cc_3
+  - pip:
+    - amply==0.1.4
+    - appdirs==1.4.4
+    - attrs==20.3.0
+    - configargparse==1.3
+    - data-model==1.2.8
+    - datrie==0.8.2
+    - docutils==0.16
+    - filelock==3.0.12
+    - gitdb==4.0.5
+    - gitpython==3.1.14
+    - importlib-metadata==3.7.0
+    - ipython-genutils==0.2.0
+    - iso8601==0.1.14
+    - jsonschema==3.2.0
+    - jupyter-core==4.7.1
+    - nbformat==5.1.2
+    - networkx==2.5
+    - obonet==0.2.6
+    - ontologyutils==0.4
+    - ontoma==0.0.17
+    - psutil==5.8.0
+    - pulp==2.4
+    - pyrsistent==0.17.3
+    - pyyaml==5.4.1
+    - ratelimiter==1.2.0.post0
+    - smart-open==4.2.0
+    - smmap==3.0.5
+    - snakemake==6.0.0
+    - toposort==1.6
+    - traitlets==5.0.5
+    - typing-extensions==3.7.4.3
+    - wrapt==1.12.1
+    - zipp==3.4.0
+prefix: /Users/irene/miniconda3/envs/snakemake
```

Comment:
I'm assuming the path from your home directory shouldn't be in the repository :)